Tools/ApplicationDebugger/array-transform/src/selector.hpp

```diff
@@ -62,3 +42,21 @@ static info::device_type GetDeviceType(int argc, char* argv[]) {
 
   return type;
 }
+
+// Return the device based on the program arguments.
+
+static device GetDevice(int argc, char* argv[]) {
+  info::device_type type = GetDeviceType(argc, argv);
+  vector<device> devices = device::get_devices(type);
+  if (type == info::device_type::host)
```

Comment:
Isn't the 'host' type deprecated?

Reply:
Compiles with 2023.0 icpx with no warnings so it doesn't look like it is deprecated.